_posts/2020-04-16-remote-junior-developer-tips.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: 'Tips From One Remote Junior Developer to Another.'
+date: 2020-04-19 10:30:00
+categories: ["junior", "remote"]
+author: fionaDL
+---
+
+Becoming the Apprentice, and then Junior Developer at OmbuLabs was both exciting and nerve racking. Learning a new company can be challenging, especially when the skills are new and the work is remote.
+
+Fortunately OmbuLabs made the journey a smooth and easy process. Here are some of the things that have been working.
+
+## Communication, communication, communication.
+
+Communication is sometimes the hardest skill to excel in when working remotely. Here at OmbuLabs we use a variety of tools and methods to help make communication easier and more accessible.
+
+### Slack:
+Communicating on Slack throughout the day is the quickest way to reach the team. As a Junior Developer it can be scary to post code and ask for help. You are constantly walking a fine line between figuring things out on your own so as not to take too much senior developer time, and knowing when it would be faster to ask for help.
+
+A great way to conquer this is to set time limits and guidelines for yourself. When you come up against an error or general topic that you don’t understand, give yourself X amount of time to decipher it. Set a timer, and when that time is up it’s the moment to turn to Slack.
+
+We have many channels on Slack. Choosing the appropriate channel helps to let the right set of Developers know that you have a question. For example, we have a general “learning” channel, but also separate channels for each project.
+
+Sometimes you can just choose a specific Senior Developer to ask the question to. Maybe the Senior Developer has helped you with that area in the past, and you won’t have to re-explain the problem, therefore it can be a quick question.
+
+### Pivotal Tracker:
+We use pivotal tracker to keep track of user stories, and set sprints for the week. Pivotal Tracker also allows for comments and this has become a useful tool when keeping track of what’s been worked on in a story with a Senior Developer.
```

Comment:
@FionaDL It would be nice to have a link to https://www.ombulabs.com/blog/agile/pivotal-tracker/how-we-use-pivotal-tracker-at-ombu-labs.html in this section

```suggestion
We use [Pivotal Tracker]( https://www.ombulabs.com/blog/agile/pivotal-tracker/how-we-use-pivotal-tracker-at-ombu-labs.html) to keep track of user stories, and set sprints for the week. Pivotal Tracker also allows for comments and this has become a useful tool when keeping track of what’s been worked on in a story with a Senior Developer.
```